src/cli/CleanFaresCommand.ts

```diff
@@ -44,6 +44,8 @@ export class CleanFaresCommand implements CLICommand {
 
   private readonly indexes = [
     "CREATE INDEX fare_flow_id ON fare (flow_id)",
+    "ALTER TABLE toc_specific_ticket ADD COLUMN IF NOT EXISTS hash VARCHAR(255); UPDATE toc_specific_ticket SET hash=COALESCE(ticket_code, restriction_code, restriction_flag, direction, end_date);",
+    "CREATE INDEX toc_specific_ticket_toc_key ON toc_specific_ticket (hash)"
```

Comment:
You probably need some procedure that will update this hash on INSERT and UPDATE

Reply:
Take a look at this construct:

https://gitlab.com/assertis/order-service/-/blob/master/migrations/20190523093505_indexes_for_leg_loading.php#L19

Reply:
No need to do it. 

1) We do the update of fares all Inserts, updates and removal
2) We run CleanFaresCommand (so after data update we'll always update toc_specific_ticket)

Reply:
Ok then go for it ;)